Marsiple/Common/Network/MartianApi.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  MatrianApi.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 18/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import Foundation
+
+struct MartianApi {
+    static let url = "https://demo.martian.agency/api"
+    
+    struct Headers {
+        static let contentType = "Content-Type"
+        static let contentTypeValue = "application/json"
+        static let xAuth = "X-Auth"
+        static let xAuthValue = "bWFydGlhbmFuZG1hY2hpbmU="
+    }
+    struct Methods {
```

Comment:
Lets add a newline here and model the different `HTTP methods` with a standalone `HTTPRequestMethod` `struct` or `enum`. 

There's no reason to attach the `Methods` to the `MartianApi`, as the entity's not exclusive tied to the `MartianApi`.

Same goes for the `Headers`.